frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -341,6 +307,39 @@ export const dashboardLogic = kea({
             }
             actions.updateDashboard(filters)
             dashboardItemsModel.actions.refreshAllDashboardItems(filters)
+            eventUsageLogic.actions.reportDashboardDateRangeChanged(filters.date_from, filters.date_to)
+        },
+        setIsOnEditMode: ({ isOnEditMode, source }) => {
+            if (isOnEditMode) {
+                clearDOMTextSelection()
+                window.setTimeout(clearDOMTextSelection, 200)
+                window.setTimeout(clearDOMTextSelection, 1000)
```

Comment:
is there a cleaner way to do this?

Reply:
I think there is, tbh I'm not entirely sure why we're doing this. Though, this PR is not changing the behavior (copied over). To move forward I would love if someone with more context on this could help us determine how to improve it.